src/main/java/com/bobocode/nasapicturesstealer/persistance/entity/Picture.java

```diff
@@ -0,0 +1,42 @@
+package com.bobocode.nasapicturesstealer.persistance.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import static jakarta.persistence.GenerationType.IDENTITY;
+
+@Data
```

Comment:
@Data include NoArgs and AllArgs. What is goal for using Builder? Do this entity will change frequently or ever?

Reply:
I used Builder in the service. So I removed it.